vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
```

Comment:
I still feel like we need concrete/abstract methods because there will be a time this burns us. Tree location doesn't guarantee it's abstractness. Maybe it's just me though. I have to translate leaf to concrete in my head every time.